app/routes/overview/components/Overview.tsx

```diff
@@ -131,6 +143,12 @@ const Overview = () => {
           <Icon onClick={showMore} name="chevron-down-outline" size={30} />
         </div>
       )}
+
+      {frontpage.length > 8 && events.length < eventsToShow && (
+        <div className={styles.showMore}>
+          <Icon onClick={scrollToTop} name="chevron-up-outline" size={30} />
+        </div>
+      )}
```

Comment:
You can merge these two checks into a single one using the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator).